### ISSUE TITLE
String interpolation colorization issue with adjacent quotes

### ISSUE THREAD

**Issue by NinoFloris** [JetBrains/resharper-fsharp#292]
Given code like:
```fs
$"""At key: "{key}""""
// all of this is now part of the string span 
```

Workaround is to add a space between the last three quotes and the inner quote
```fs
$"""At key: "{key}" """
// all of this is all good again
```

This issue (I reported a duplicate) seems related, but at least you have a repro now :)
https://youtrack.jetbrains.com/issue/DEXP-568210

**Comment by auduchinok** [comment]
@NinoFloris Very nice case, thanks!

I'm looking into it and other interpolation-related issues now.

**Comment by auduchinok** [comment]
OK, this one is fixed in [76161ec](https://github.com/JetBrains/resharper-fsharp/commit/76161eccf62a1bc09fa1070d351cac00dee0edce).

There're some cases related to https://github.com/JetBrains/resharper-fsharp/issues/263 and https://github.com/JetBrains/resharper-fsharp/issues/282 to fix, though. I'm on it.
